experimental/framework/frs-client/src/test/CreateClient.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { generateUser } from "@fluidframework/server-services-client";
+import {
+    FrsClient,
+    FrsConnectionConfig,
+    InsecureTokenProvider,
+} from "..";
+
+export class CreateClient {
```

Comment:
CreateClient ? It's really hard to even imaging what this class does based on the name. Does it help to create client? What kind of client (maybe it's fine to omit given package name, but we do put "Fluid" everywhere, so I'd do same with Frs).

This looks like FrsClientFactory to me based on quick glance.

Consider adding good comment about class describing purpose of this class.

Reply:
It would be great to name file appropriately as well

Reply:
Thanks for the pointer! 😃

Reply:
this could exist as just a function too, this was more to refactor some setup specific stuff out of the tests

Reply:
a function inside the test file? or in the current `CreateClient.ts` (will be renamed to match the function)?

Reply:
a single function in this file

Reply:
changed the file name to `FrsClientFactory` with a single function `createFrsClient()`. But if we are moving away from having environment variable and have dedicated test files instead for local and remote mode respectively, then maybe this file is unnecessary.